saleor/graphql/product/mutations/collection/collection_create.py

```diff
@@ -111,14 +112,23 @@ def clean_input(cls, info: ResolveInfo, instance, data, **kwargs):
         clean_seo_fields(cleaned_input)
         return cleaned_input
 
+    @staticmethod
+    def batch_product_ids(ids):
+        # Batch size of 25k ids, assuming their pks are at least 7 digits each
+        # after json serialization, weights 225kB of payload.
+        BATCH_SIZE = 25000
+        _length = len(ids)
+        for i in range(0, _length, BATCH_SIZE):
+            yield ids[i : min(i + BATCH_SIZE, _length)]
```

Comment:
Do you need a `min`  here?
```
ids = [9,8,7,6,5,4,3,2,1]

In [17]: def a(ids):
    ...:     BATCH_SIZE = 2
    ...:     _length = len(ids)
    ...:     for i in range(0, _length, BATCH_SIZE):
    ...:         yield ids[i : i + BATCH_SIZE]
    ...:

In [18]: print([q for q in a(ids)])
[[9, 8], [7, 6], [5, 4], [3, 2], [1]]


In [20]: ll = [1,2,3]

In [21]: ll[0:1000]
Out[21]: [1, 2, 3]
```

PS. (if not needed, I think it is ok, to only change it for `main` branch)